src/Exception/RequestException.php

```diff
@@ -102,13 +106,13 @@ public static function create(
             $response->getReasonPhrase()
         );
 
-        $summary = \GuzzleHttp\Psr7\Message::bodySummary($response);
+        $summary = ($bodySummarizer ?? new BodySummarizer())->summarize($response);
```

Comment:
I agree with this line because it keeps the same behavior. 

In Guzzle8 I would like to remove this fallback. If you don't provide a `BodySummarizerInterface` then you will not get a summary.

Reply:
Oh. In which case I'd advocate for changing the parameter order, perhaps, or maybe providing more than one create method. Like `createWithException` or something.